### ISSUE TITLE
test loss and training loss fluctuating up and down rapidly 

### ISSUE THREAD

**Issue by 292550** [tensorflow/playground#164]
![screenshot2](https://user-images.githubusercontent.com/96833601/147678657-cb9158f7-daa9-4e0b-9cdf-56a1390de815.png)
![screenshot1](https://user-images.githubusercontent.com/96833601/147678659-e748d424-4866-4a2d-9568-a746296ccac6.png)
